### PR TITLE
add tracking limit order + update ux limit order

### PR DESCRIPTION
**Pull request by nguyenhoaidanh**
- add tracking limit order
- fix bug can not verify hardware wallet
- update ux limit order

### CODE REVIEW

File: src/index.tsx
Comment (namgold) [inline src/index.tsx:90]: remove in `react-app-env.d.ts` too

File: src/components/swapv2/LimitOrder/useBaseTradeInfo.ts
Comment (namgold) [inline src/components/swapv2/LimitOrder/useBaseTradeInfo.ts:50]: May crash. Should wrap a try catch?

Comment (nguyenhoaidanh) [reply src/components/swapv2/LimitOrder/useBaseTradeInfo.ts:50]: no need, already added try catch in useSWR

File: src/index.tsx
Comment (namgold) [inline src/index.tsx:106]: This should be a separated PR. This need go to main soon, we need it too

Comment (nguyenhoaidanh) [reply src/index.tsx:106]: I will merge this pr now after your approval. 🙏

Comment (namgold) [reply src/index.tsx:106]: ah sorry I though that we are merging to another branch :))

File: src/components/swapv2/LimitOrder/LimitOrderForm.tsx
Comment (namgold) [inline src/components/swapv2/LimitOrder/LimitOrderForm.tsx:467]: tested?

Comment (nguyenhoaidanh) [reply src/components/swapv2/LimitOrder/LimitOrderForm.tsx:467]: yes, QC test sấp mặt.

**Review by namgold — APPROVED** [review]

**Pull request merged**
